src/app/components/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Component, OnInit } from '@angular/core';
 import { Router, RoutesRecognized } from '@angular/router';
-import { BreadcrumbInterface } from '../interfaces/breadcrumb-interface';
+import { BreadcrumbInterface } from '../../shared/interfaces/breadcrumb-interface';
```

Comment:
Please create [paths ](https://stackoverflow.com/questions/43281741/how-to-use-paths-in-tsconfig-json)for mostly used folders in tsconfig.json. 
After that imports would be shown as 
'../../shared/interfaces/breadcrumb-interface'; => '@shared/interfaces/breadcrumb-interface';